test/lock_retrier_test.rb

```diff
@@ -32,6 +32,7 @@ def teardown
   def test_with_retries
     with_table_locked(:users) do
       with_lock_retries do
+        # this doesn't pass the command to the lock_retrier.lock_timeout method - why?
```

Comment:
I'm not sure why but this test setup in a transaction doesn't pass the command  through, i believe it's due to [this code block](https://github.com/fatkodima/online_migrations/pull/68/files#diff-23f4031ffb47063838068c3baf33759abd06abcfb14634a1b8db446af98db65fR25).